### DISCUSSION TITLE
몬스터 레벨 업 관련한 상세 질문

### DISCUSSION THREAD

**Discussion by Jeon-YJ1004** [SeoulTechTCPGame/HumanSurvival#245] [Q&A]
[몬스터 레벨 위키](https://github.com/SUST22Unity/HumanSurvival/wiki/%EB%AA%AC%EC%8A%A4%ED%84%B0-%EB%B3%84-%EC%88%98%EC%B9%98#%EC%8A%A4%ED%85%8C%EC%9D%B4%EC%A7%80-1)
위 링크의 위키에 아래 질문의 답이 모호합니다.

1. '캐릭터의 레벨이 증가함에 따라 몬스터의 레벨도 증가하여 실제 몬스터의 체력도 증가한다' 는 현재 몬스터 레벨=초기 몬스터 레벨 + 캐릭터 레벨 인건가요? 몬스터 마다 처음 스폰되는 시간이 있는데 그때 캐릭터 레벨부터의 증가량이 몬스터 레벨 증가량인건가요?
ex) 박쥐는 캐릭터 레벨 1부터 등장하니 => 캐릭터 레벨= 박쥐 레벨로 볼 수 있다.
ex) 머드맨은 5분 부터 등장하고 이때 캐릭터 레벨이 10이라면 이후 캐릭터 레벨이 11일때 머드맨 레벨은 2가 된다. 

2. 플레이어 레벨 업하면 이미 스폰된 몬스터의 체력은 관련이 없고 레벨업 이후 스폰될 몬스터의 레벨만 영향이 있는 건가요?

**Comment by rapa1130** [comment]
### 1
문서에 초기 몬스터 레벨과 관련된 구체적인 사항이 작성되어 있지 않았네요. 캐릭터 레벨과 Level 중에 더 큰 값을 몬스터 레벨 값으로 넣어야 합니다.

`몬스터 레벨 = Max( 캐릭터 레벨 , 몬스터 Level 스탯 )`

그리고  end+ level 이 해당 몬스터의 **최대 몬스터 레벨**입니다. 이보다 더 캐릭터 레벨이 올라간다고 해도 몬스터 레벨이 더 이상 증가하지 않습니다.

예시로 드신 박쥐의 경우라면 캐릭터가 1레벨이고 몬스터 레벨 스탯도 1이기 때문에 맞습니다.
머드맨의 경우엔 캐릭터 레벨이 11이 된다면 몬스터 레벨 스탯이 1이기 때문에 몬스터 레벨은 11이 됩니다.

### 2
맞습니다


설명이 부족했거나 이해가 안되신 부분이 있다면 질문 바랍니다.
